src/resources/startup.py

```diff
@@ -56,14 +56,17 @@ def serialize_byte_tensor(input_tensor):
         Serializes a bytes tensor into a flat numpy array of length prepend bytes.
         Can pass bytes tensor as numpy array of bytes with dtype of np.bytes_,
```

Comment:
When need to enforce (or at least educate) users to use np.object for their BYTES tensors and point explicitly to the problem with np.bytes_ and np.str. Even though these are internal we should be clear in what proper usage is.

Reply:
I improved this docstring.